op-chain-ops/cmd/op-migrate/main.go

```diff
@@ -164,10 +164,7 @@ func main() {
 
 			dbCache := ctx.Int("db-cache")
 			dbHandles := ctx.Int("db-handles")
-
-			chaindataPath := filepath.Join(ctx.String("db-path"), "geth", "chaindata")
-			ancientPath := filepath.Join(chaindataPath, "ancient")
-			ldb, err := rawdb.NewLevelDBDatabaseWithFreezer(chaindataPath, dbCache, dbHandles, ancientPath, "", false)
+			ldb, err := db.Open(ctx.String("db-path"), dbCache, dbHandles)
```

Comment:
`db-path` doesn't have `Required: true` set so this can be an empty string. Perhaps we want to check for empty values inside of `db.Open` and return an error? Also could add `Required: true` to cli args in this script, could be a diff PR